src/components/Editor.jsx

```diff
@@ -129,9 +129,27 @@ class Editor extends React.Component {
 
   _startNewSession(source) {
     const session = createAceSessionWithoutWorker(this.props.language, source);
+    const cursor = session.selection.lead;
```

Comment:
Seems like `lead` as a property may not be part of the public API—the [docs](https://ace.c9.io/#nav=api&api=selection) only mention `getSelectionLead()`